Entity/PageManager.php

```diff
@@ -118,6 +118,40 @@ public function findOneByUrl($site, $url)
         return null;
     }
 
+    /**
+     * Test the $page if it matches the $url
+     *
+     * @param Page $page
+     * @param $url
+     * @return bool
+     */
+    protected function matches(Page $page, $url)
+    {
+        $purl = $page->getUrl();
+        $pattern = '#^'.$purl.'$#';
+        preg_match_all('/{[a-z]+}/', $pattern, $matches);
+
+        if ($tokens = $matches[0]) {
```

Comment:
Unnecessary test. $matches should be always an array. (Side note: Assignment as only operation of condition is considered as bad practice by majority of code standards.)